# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/quota/QuotaManager.h
# Commit: b12a2f55233f
# Full Hash: b12a2f55233fade8f2e3411295ff3d362df9e359
# Author: Jan Varga <jvarga@mozilla.com>
# Date: 2024-10-09 21:46:09
# Regressor Bug: 1866402
# File Overlap Count: 1
# Description:
#   Bug 1866402 - Change nsIQuotaManagerService::ResetStoragesForPrincipal to support resetting of entire origins only; r=dom-storage-reviewers,jari
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D195510
# ==============================================================================

diff -r 6a90f36c8bf5 -r b12a2f55233f dom/quota/QuotaManager.h
--- a/dom/quota/QuotaManager.h	Wed Oct 09 11:59:34 2024 +0000
+++ b/dom/quota/QuotaManager.h	Wed Oct 09 11:59:35 2024 +0000
@@ -457,7 +457,7 @@
 
   RefPtr<BoolPromise> ShutdownStoragesForOrigin(
       Maybe<PersistenceType> aPersistenceType,
-      const PrincipalInfo& aPrincipalInfo, Maybe<Client::Type> aClientType);
+      const PrincipalInfo& aPrincipalInfo);
 
   RefPtr<BoolPromise> ShutdownStoragesForClient(
       Maybe<PersistenceType> aPersistenceType,